control/Director.php

```diff
@@ -422,13 +422,23 @@ public static function set_current_page($page) {
 	 */
 	public static function absoluteURL($url, $relativeToSiteBase = false) {
 		if(!isset($_SERVER['REQUEST_URI'])) return false;
-		
+
+		//a url of . or ./ is the same as an empty url
+/*
+		if ($url == '.' || $url == './') {
```

Comment:
Don't need to bracket short conditions. `if($url === '.' || $url === './') $url = '';`